Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -60,8 +61,21 @@ protected function makeTest($expected, $input = null, \SplFileInfo $file = null,
             $this->assertSame($expected, $tokens->generateCode(), 'Code build on input code must match expected code.');
 
             Tokens::clearCache();
+            $expectedTokens = Tokens::fromCode($expected);
             $tokens->clearEmptyTokens();
-            $this->assertTokens(Tokens::fromCode($expected), $tokens);
+            $this->assertTokens($expectedTokens, $tokens);
+
+            $tokensReflection = new \ReflectionClass($expectedTokens);
+            $propertyReflection = $tokensReflection->getProperty('foundTokenKinds');
+            $propertyReflection->setAccessible(true);
+            $foundTokenKinds = array_keys($propertyReflection->getValue($expectedTokens));
```

Comment:
I get all kinds from `expectedTokens` and check if there are present in `tokens` (the fixed one). So check what should be in what we have.

There is no need to do this inside `assertTokens` - now we made only one assertion for each token kind, in `assertTokens` we will made it for every token, even if we already test that token kind.